peer/log.go

```diff
@@ -133,6 +148,23 @@ func messageSummary(msg wire.Message) string {
 		return fmt.Sprintf("hash %s, ver %d, %d tx, %s", msg.BlockHash(),
 			header.Version, len(msg.Transactions), header.Timestamp)
 
+	case *wire.MsgMixPairReq:
+		return mixMessageSummary(msg)
+	case *wire.MsgMixKeyExchange:
+		return mixMessageSummary(msg)
```

Comment:
These are definitely an improvement over the lack of anything that is there now, but I think we should put a few more details in the `!MsgMixPairReq` cases.

For example, maybe the run number and session id would be nice.

Then, for `MsgMixCiphertexts` perhaps the number of ciphertexts and `MsgMixDCNet` the number of mix vectors, etc.

Reply:
session/run we can do, the rest will make the "summary" just way too verbose i think. after adding the session we are already at 2 hashes and a serialized pubkey.

Reply:
Fair point.  They're pretty long.  Looks better with the session and run, imo.  I already say a `run 1` in my logs with this update.